kat/src/lms_kat.rs

```diff
@@ -0,0 +1,278 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    lms_kat.rs
+
+Abstract:
+
+    File contains the Known Answer Tests (KAT) for LMS cryptography operations.
+
+--*/
+
+use crate::caliptra_err_def;
+use caliptra_drivers::{
+    CaliptraResult, HashValue, LmotsAlgorithmType, LmotsSignature, Lms, LmsAlgorithmType,
+    LmsIdentifier, LmsSignature,
+};
+
+caliptra_err_def! {
+    LmsKat,
+    LmsKatErr
+    {
+        DigestFailure = 0x01,
+        DigestMismatch = 0x2,
+    }
+}
+
+#[derive(Default, Debug)]
+pub struct LmsKat {}
+
+impl LmsKat {
+    /// This function executes the Known Answer Tests (aka KAT) for LMS.
+    pub fn execute(&self, lms: &Lms) -> CaliptraResult<()> {
+        self.kat_lms_24(lms)
+    }
+
+    fn _kat_lms_24(&self, _lms: &Lms) -> CaliptraResult<()> {
```

Comment:
Is this intended to be used?

Reply:
No, it was left in there as a stub to essentially disable the LMS KAT since including the KAT code puts us right at the ROM size limit.  It can be removed if we think we are safely below the ROM limit, or people could remove the LMS kat in their on branches if they are hitting this limit.

Reply:
Will remove